### PR TITLE
Improvments for faster use

### PR DESCRIPTION
**Pull request by lgvr123**
* New option to select the full score or the current measure (original)in case of a single note selection
* New option to have a strict or unstrict (original) counting of the notes 
* Split into 3 plugin 
    * Up: selection from bottom, 
    * Down: selection from top, 
    * Original: original behaviour

### CODE REVIEW

File: FilterSimilarNoteLevels/core.js [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+* @param expandToFullScore true(default)|false 
+*/
+function makeSelection(up, forceFullScore, strictCounting, expandToFullScore) {
+
+    /// choose counting direction /////
+    var counting = (up)?"up":"down";
+    /// expand single selection to score or measure ? ///
+    var defaultExpand=((typeof expandToFullScore === "undefined")||expandToFullScore)?"score":"measure";
+    var strict=(strictCounting)?true:false;
+    var fullScore=(fullScore)?true:false;
```
Comment (Ash-86) [inline FilterSimilarNoteLevels/core.js]: you mean     
    var fullscore=(forceFullScore)?true:false  ?

but i noted you did not assign this parameter as true in any file.

**Comment by Ash-86** [comment]
Hi, sorry for the late reply as i was traveling and just got back. I'm glad you found the plugin useful enough to want to propose improvements. 

I see you prefer strict counting. I actually had written the plugin to count strictly but changed it because folks on the forum preferred to have the ability to select top and bottom notes. 

You also prefer to have it directly select the fullscore. I assume you run the plugin from the menu ? I map the shortcut to "z", for a quick fullscore selection just press z twice.

**Comment by lgvr123** [comment]
Your plugin is really useful. I, once, considered building an "Interactive" UI where you could specify your preferred options and save them in the settings (I did that for my [SoloAnalyser](https://github.com/lgvr123/musescore-soloanalyser) plugin)

I will check your remark about "fullScore". If needed I'll make another Pull Request.

**Comment by Ash-86** [comment]
No need for another pr, I fixed the variable name in the last commit.  It now works as you intended: to forceFullScore no matter the selection type.  (though i think this option is a bit superfluous)
An interactive UI for selecting the preferred setting sounds like a nice idea if you really are into that:) 
Thank you, Laurent!

**Pull request merged**